webui/framework/src/main/java/org/exoplatform/webui/event/Event.java

```diff
@@ -99,6 +99,8 @@ public final void broadcast() throws Exception {
                        this.getName(),
                        listener.getClass().getName());
             }
+            //Avoid the display of a blank page
+            context_.getJavascriptManager().getRequireJS().addScripts("location.reload();");
```

Comment:
@hakermi @AzmiTouil Could you revert this change, instead could you throw an exception that you can use in the upper application level to refresh page or just refresh the CSRF token ?